CMakeLists.txt

```diff
@@ -271,7 +271,7 @@ if(COMPILE_CPU)
     find_package(BLAS)
     if(BLAS_FOUND)
       include_directories(${BLAS_INCLUDE_DIR})
-      set(EXT_LIBS ${EXT_LIBS} ${BLAS_LIBRARIES})
+      set(EXT_LIBS ${EXT_LIBS} ${BLAS_LIBRARIES} cblas)
```

Comment:
This should rather be done by the typical Find* semantics of CMake. I don't think is clear at all that libcblas exists at this point?.

Reply:
Shouldn't "cblas" appear inside ${BLAS_LIBRARIES} if present?

Reply:
I guess FindBLAS finds the fortran library, not the C library:
```
# This module finds an installed fortran library that implements the
# BLAS linear-algebra interface (see http://www.netlib.org/blas/).  The
# list of libraries searched for is taken from the autoconf macro file,
# acx_blas.m4 (distributed at
# http://ac-archive.sourceforge.net/ac-archive/acx_blas.html).
```
So I guess to be pedantic, it's within its rights to not provide the cblas library for C. 

cblas is part of OpenBLAS: https://github.com/xianyi/OpenBLAS/issues/1297

CMake doesn't have an included FindCBLAS or FindOpenBLAS but both of these are floating around other people's projects.

Reply:
So, are we doing anything here? The current solution does fine OpenBLAS for me, at least on Ubuntu.